src/mon/Paxos.cc

```diff
@@ -923,7 +923,7 @@ void Paxos::commit_finish()
   new_value.clear();
 
   // WRITING -> REFRESH
-  // among other things, this lets do_refresh() -> mon->bootstrap() know
+  // among other things, this lets do_refresh() -> mon->bootstrap() now
```

Comment:
I don't understand this comment *either* way around off the top of my head. I think it's saying that the do_refresh() to mon->bootstrap() transition doesn't need to flush the store queue? So we're telling it that by changing states? In which case "know" is correct. But perhaps something else is happening here.

Reply:
i think `do_refresh() -> mon->bootstrap()` is not a transition from `do_refresh()` to `mon->bootstrap()`, it means `do_refrehs()` calls `mon->bootstrap()`.